app/src/main/java/org/wikipedia/Constants.java

```diff
@@ -75,6 +75,7 @@
         FEED_CARD_SUGGESTED_EDITS_TRANSLATE_DESC,
         SUGGESTED_EDITS_ONBOARDING,
         ONBOARDING_DIALOG,
+        APP_SHORTCUTS,
```

Comment:
Can we add new values at the end of the list, so that the ordinal numbers wouldn't be reshuffled every time we refactor these parts?

Reply:
Done. Will adjust https://github.com/wikimedia/apps-android-wikipedia/pull/200/files after this is merged.